results.php

```diff
@@ -11,6 +12,17 @@
       <a id='nav_browse' href='browse.html'>Browse</a>
     </div>
     <div style='padding:1rem;'></div>
+    <div style='padding:1.5rem;'></div>
+    <select>
+      <option value=-1>All</option>
+      <option value=1>Photos</option>
+      <option value=2>Videos</option>
+      <option value=3>Audio Recordings</option>
```

Comment:
It looks like the selection UI for document type is a drop-down. I recommended initially that we use tabs. Is there a particular reason the choice was made to use a drop-down instead?

Reply:
I was imagining something like this: https://www.w3schools.com/howto/howto_js_tabs.asp. Bootstrap also has some nice ways to do this: https://www.w3schools.com/bootstrap/bootstrap_tabs_pills.asp. This could be something we leave for UI work later... after all, we only really care about getting the bare functionality needed to do the work to connect the website to the database in the next sprint. I'll leave it up to you to change to tabs or merge.